### PR TITLE
Fixed padding issues with the author information.

### PR DESCRIPTION
**Pull request by VSB14**
# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details:
- The Padding attributes "px-4" have been removed from Biobar.tsx file to fix the alignment issues with the author information at the end of each article.
## Any Breaking changes:
- None
## Associated Screenshots:
    Before Commit
image

    After Commit
image

### CODE REVIEW

File: components/BioBar/BioBar.tsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ const BioFooter = ({ author }: Props) => {
 
   const { name, image, bio, username } = author;
   return (
-    <div className="max-w-xl px-4 mx-auto  text-gray-700 dark:text-gray-300 mt-6">
-      <div className="flex mx-2 sm:mx-6 md:mx-auto px-4 border-t-2 pt-6 border-gray-300 dark:border-gray-800">
+    <div className="max-w-xl mx-auto  text-gray-700 dark:text-gray-300 mt-6">
```
Comment (auxfuse) [inline components/BioBar/BioBar.tsx:18]: Could we consider removing the double-space present in this line between `mx-auto` & `text-gray-700` ? 🤔 It's not a big deal that won't break anything, but it is a nice standard to adhere to 😅

**Review by auxfuse — CHANGES_REQUESTED** [review]
Nice change here! Just one thing for feedback considerations 🤜

**Comment by pkspyder007** [comment]
@auxfuse I think we should add linting config and git hooks to automate these small issues.
can you open some issues for the same, It would be some nice beginners issues

File: components/BioBar/BioBar.tsx (hunk 1 above)
Comment (NiallJoeMaher) [reply components/BioBar/BioBar.tsx:18]: Nice catch! Eagle eyes 🦅

**Pull request closed without merging**